# The camera intrinsic parameters matrix
# 3x3 matrix
camera_intrinsic_parameters: [700, 0, 960, 0, 700, 540, 0, 0, 1]

R: [0, 0, -1, 1, 0, 0, 0, 1, 0]

# Sampling time
dt: 0.01

seguimiento:
  len: 4
  data: [96, 97, 98, 99]

# Gain on linear velocities
gainv: 3
gainv_max: 5
gainv_prime: 2

# Gain on angular velocities
gainw: 0.5
gainw_max: 0.5001
gainw_prime: 1

# Threshold on the feature position error, used to stop the visual control
feature_error_threshold: .01

nfeatures: 1500
scaleFactor: 1.2
nlevels: 8
edgeThreshold: 15
patchSize: 30
fastThreshold: 20
flann_ratio: 0.7

############################################################
# Cambio de control 1 (1/dist) o 2 (dist).
control: 1

camara: 1

# THIS CONTROL DOES NOT USE THE FOLLOWING PARAMETERS BUT THEY ARE REQUIRED
gainv_ki: 0
gainv_ki_max: 0.01
gainv_ki_prime: 1
